### DISCUSSION TITLE
Add issue/task templates to GitHub projects

### DISCUSSION THREAD

**Discussion by Zerotask** [community/community#63468] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

Currently you can define awesome [issue forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms) with issue templates.
It would be great, if you could use them in your linked project too.

In our use case, we have 1 repository connected to 1 project. Users have 2 options to add something to the project:
- create an issue in the repository, which then will be linked to the project (workflow automation or issue template projects property)
- create a task in the project and convert it to an issue

Since we don't have simple task, but always issues, it would be nice, to have the same form for both.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by togishima** [comment]
I was looking for a way to add issue templates to projects and came across to this conversation...
I assume there is no way to add issue templates to projects as of yet, but would be nice like mentioned above to be able to have same template for when drafing/creating issue from projects

**Comment by Julian-Dumitrascu** [comment]
Nor do we need the type of data set called "tasks".
So I'd like us to discuss allowing people to use "issues" instead of "tasks".
Until you create the necessary feature, how can one have GitHub convert each task to an issue automatically once a task is created?

We don't need "discussions" either. They seem a poorer version of "issues". So turn this message into an issue and let's vote on focusing on issues!
